packages/grid/_modules_/grid/models/gridExport.ts

```diff
@@ -1,12 +1,24 @@
+export interface GridExportCsvOptions {
+  utf8WithBom?: boolean;
```

Comment:
Why only have this option with the CSV format?

Reply:
In the future there would be a delimiter option or even a decimal separator.